### ISSUE TITLE
[TASK] Assess content needs for CHAMPVA beneficiaries and others who use Meds by Mail

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#67895]
CHAMPVA and other Meds by Mail users have a different experience in the medications tool, because we don't have their full medical records and they can't access certain parts of the health tools, like secure messaging.

We need to assess how these users currently experience the MHV tools, and collaborate on content updates to medications and other tools to accommodate these users in phase 1. This may involved dynamic tailored content that displays only to these users.

**Comment by kfink24** [comment]
No we won't add a note to the Try me banner because it becomes quite long and the experience is equivalent to what they have today on MHV.

**Comment by mistymg** [comment]
@mistymg check with Randi to see if she took care of this

**Comment by RLHecht** [comment]
Thanks @mistymg. I wasn't involved with this but it seems like from Laura's comment that the Meds by Mail info was already added to the tool so I think this is good to close. I would just validate first with a test user that this text is in there.

**Comment by sterkenburgsara** [comment]
I think my content question for you would be: what is the information that we provide to Meds by Mail users, and do we think they will understand 1.) they can request mediation refills via My HealtheVet on VA.gov, but cannot use the other tools? 2.) Know where to take their questions if they have them? 

Where should the answer to those questions (and the content that was previously in that accordion element on teh old landing page) live? Right now, I see content under the "for families" heading on [this unauthenticated page](https://staging.va.gov/health-care/review-medical-records/), but it doesn't include the link to the "find your meds by mail servicing center" that was previously listed in tool landing pages. Is that okay? Do we have feedback from users that they are looking for this information but can't find it? 

You may want to reach out to Robyn Singleton and/or Kay Lawyer (OCTO health stakeholders) about this to see if they have thoughts.

**Comment by sterkenburgsara** [comment]
cc @rbsingleton @klawyer

**Comment by rbsingleton** [comment]
In terms of the larger question re: how best to communicate what to Meds by Mail users, that probably requires more discovery that we're not currently resourced to prioritize. Happy to turn it into a ticket to hang in teh backlog until someone can get around to it though. 

For this specific content, the thing that would probably make the most sense would be to have it be triggered if any of the Meds by Mail users (who we can identify now) clicks on the "Go to your allergies and reactions" link on the Meds list page. That being said, there is some content there for Meds by Mail users about calling their servicing center. 

So tl;dr - interested to hear what come out of @mistymg  and @VCM86 's working session, and while this isn't the most pressing priority for meds at the moment, it wouldn't hurt to keep track of the thinking and convos for when we can get to it. @sterkenburgsara 

<img width="692" height="841" alt="Image" src="https://github.com/user-attachments/assets/2232b7b8-9234-43c8-a4c1-fbffa4013b8b" />

**Comment by VCM86** [comment]
Capturing some initial thoughts here:

We plan to run this ticket by Danielle in a working session next week as well, but my thought now is that there are 2 clear options. We could add a small amount of information to the current paragraph under the "Medications" heading--maybe just the "Find your Meds by Mail servicing center" link.

The 2nd option would be to include all or most of this previous content as an informational alert, possibly triggered dynamically when Meds by Mail users click on the "Go to your allergies and reactions" link.

<img width="606" height="744" alt="Image" src="https://github.com/user-attachments/assets/85df7ca7-d882-4c0a-9d94-03517f95463f" />

**Comment by VCM86** [comment]
@rbsingleton & @sterkenburgsara -- After chatting with Danielle, we think the best course of action would be these 2 steps: 

- Remove this sentence from the **Medications** page: "If you use Meds by Mail, you can also call your servicing center and ask them to update your records."
- Include this previous content as an informational alert triggered dynamically when Meds by Mail users click on the "Go to your allergies and reactions" link.
<img width="606" height="744" alt="Image" src="https://github.com/user-attachments/assets/7b628cfb-7cab-4224-9124-2e43bc9b9ea6" />

Can you let me know what the next steps would be for this, and if the content team would need to provide more support for the alert? Thank you!

**Comment by michaelbrodsky-a6** [comment]
Thanks all! Just a very quick follow-up to let you know that the Medications team is now aware of this work. I'll ticket this shortly, bring it up with the engineers, and reach back out if we have any implementation-related questions. Really appreciate you documenting everything so helpfully here.

**Comment by VCM86** [comment]
Perfect--thanks @michaelbrodsky-a6! I'll go ahead and close this ticket, but please reach back out if you have any questions or need other content support. :-)

**Comment by michaelbrodsky-a6** [comment]
Thanks so much @VCM86 – sounds great. Really appreciate it.

**Comment by VCM86** [comment]
Hi @sterkenburgsara & @rbsingleton -- I've been doing a little sleuthing to try to determine where that link to the Meds by Mail servicing centers in the original content rec was meant to go, but I'm having trouble finding that info. Do either of you have any memory or context--or know anyone who might know more?

**Comment by VCM86** [comment]
Reopening this ticket until we determine the link location.

**Comment by sterkenburgsara** [comment]
@VCM86 see Slack conversation [here](https://dsva.slack.com/archives/C04PRFEJQTY/p1754058136333169).

**Comment by VCM86** [comment]
That's great @michaelbrodsky-a6! Please let me know if you all will need more content support or if we can go ahead and close out this ticket. :-)

**Comment by VCM86** [comment]
Thanks so much @michaelbrodsky-a6! Happy to help, and closing this ticket out now.
